<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mummery</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/new/prefix1/resources/icons8-mask-60.png</normaloff>:/new/prefix1/resources/icons8-mask-60.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_inFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Входной файл</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_inFile">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Например: file.txt</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Имя файла</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_deleteInFile">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Удаление после преобразования</string>
     </property>
     <property name="text">
      <string>Удалять входной файл</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_otFile">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>371</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Выходной файл</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_otFilePath">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Полный путь</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_rewrite">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>При наличии одноименного файла</string>
     </property>
     <property name="text">
      <string>Перезапись</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_copying">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Копия</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_transform">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Преобразовать</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_transform">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>951</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Параметры преобразования</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_8BMask">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>В шестнадцатиричной системе</string>
     </property>
     <property name="inputMask">
      <string>HHHHHHHHHHHHHHHH</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Маска размером 8 байт</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_timer">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>50</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Работа по таймеру</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_operation">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Например: XOR, AND и т.д.</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Поразрядная операция</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_interval">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>100</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;В секундах. Для указания в милисекундах укажите постфикс ms. Например, 500ms.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="placeholderText">
      <string>Интервал опроса</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>986</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Справка</string>
    </property>
    <property name="icon">
     <iconset resource="Resources.qrc">
      <normaloff>:/new/prefix2/resources/icons8-help-32.png</normaloff>:/new/prefix2/resources/icons8-help-32.png</iconset>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionf">
   <property name="text">
    <string>f</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
